pubspec.yaml

```diff
@@ -39,8 +39,9 @@ dev_dependencies:
   dart2_constant: ^1.0.0
   dart_dev: ^3.0.0
   dependency_validator: ^1.4.0
+  front_end: ^0.1.27
```

Comment:
Why was this added as a dependency?

Reply:
It definitely has to do with build_vm_compilers `1.0.3`. That version (and greater) in particular creates a build error that complains about not having front_end. I'm not sure why the dependency tree is getting into a bad state though

Reply:
Gross. I'm fine if this is necessary since it's just a dev dependency, but would you add a comment explaining this so we know that it's safe to remove down the road once it starts working again?

Reply:
Yes sir, that's a good idea